package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "scripts": {
     "dev": "parcel index.html",
+    "dev-local": "LOCAL_DATA=true parcel index.html",
```

Comment:
Set up to serve sheets data locally

TODO: Add npm command to download a given sheet to local data